templates/intro-jobs/README.ipynb

```diff
@@ -0,0 +1,112 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to Jobs\n",
+    "\n",
+    "This tutorial shows you how to:\n",
+    "1. Run a Ray app non-interactively in Anyscale as an \"Anyscale Job\".\n",
+    "2. Configure and debug Anyscale Jobs.\n",
+    "\n",
+    "**Note**: This tutorial is run within a workspace. Please overview the `Introduction to Workspaces` template first before this tutorial.\n",
+    "\n",
+    "## Key features of Anyscale Jobs\n",
+    "\n",
+    "Typically, we recommend running batch Ray apps as Anyscale Jobs when moving to production. Like workspaces, Anyscale Jobs run with their own Ray cluster, so you can run the exact same Ray program in a workspace as a Job too.\n",
+    "\n",
+    "Key features of Anyscale Jobs:\n",
+    "- Programmatic submission API\n",
+    "- Automated failure handling\n",
+    "- Automated email alerting\n",
+    "- Record and persist outputs such as logs\n",
+    "\n",
+    "\n",
+    "**Note**: Ray also has an internal concept of a \"Ray job\", which is created when running a Ray app. Anyscale Jobs, Workspaces, and Services all launch Ray jobs internally."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Walkthrough\n",
+    "\n",
+    "First, let's run the following app first interactively in the current workspace.\n",
+    "\n",
+    "This template includes a simple processing job in **./main.py** that runs a few Ray tasks. Run the cell below in the workspace, you should see it print the result after a few seconds."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# First install the necessary `emoji` dependency, then run the Ray app\n",
+    "!pip install emoji; python main.py"
```

Comment:
I would separate these into two cells. otherwise kind of weird to see their output intermingled and you might want to run `main.py` again or add some other dependencies